Dockerfile

```diff
@@ -8,6 +8,16 @@ RUN pip install poetry
 COPY ./pyproject.toml ./poetry.lock* /tmp/
 RUN poetry export -f requirements.txt --output requirements.txt --without-hashes
 
+######
```

Comment:
This ordering doesn't seem correct -- you need to put build containers before the actual build?

Or should this be called `2. ...` and the later step called `3. ...`?

Reply:
Yeah it was just mis-labeled